Actions/ReadSecrets/ReadSecrets.ps1

```diff
@@ -50,10 +75,20 @@ try {
                 }
             }
         }
+
+        if ($updateSettingsWithValues) {
+            $outSettings.appDependencyProbingPaths | 
+            ForEach-Object {
+                if ($($_.authTokenSecret)) {
+                    $_.authTokenSecret = GetGithubSecret -secretName $_.authTokenSecret 

```

Comment:
This approach requires the secrets to be in github secrets and you throw an error if it isn't.
We should do the same as with the secretscollection - take the ones we can from Github secrets and optionally the rest from the KeyVault (if defined) only if appDependencyProbingpaths secrets are not found in either - we should output an error.
I do agree that missing secrets from probing paths should emit an error (which is different from what the secret collection does)